src/components/application_manager/test/commands/mobile/reset_global_properties_test.cc

```diff
@@ -149,12 +149,23 @@ TEST_F(ResetGlobalPropertiesRequestTest, Run_InvalidVrHelp_UNSUCCESS) {
       mobile_apis::GlobalProperty::KEYBOARDPROPERTIES;
 
   EXPECT_CALL(app_mngr_, RemoveAppFromTTSGlobalPropertiesList(kConnectionKey));
-  smart_objects::SmartObject so_prompt =
+
+  std::vector<std::string> help_prompt{"help_text"};
```

Comment:
@AKalinich-Luxoft 
This is C++11 feature, is that standard already allowed for open SDL?